enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/LeaderInfo.java

```diff
@@ -39,13 +39,18 @@ public LeaderInfo( MemberId memberId, long term )
         this( memberId, term, false );
     }
 
-    public LeaderInfo( MemberId memberId, long term, boolean isSteppingDown )
+    private LeaderInfo( MemberId memberId, long term, boolean isSteppingDown )
     {
         this.memberId = memberId;
         this.term = term;
         this.isSteppingDown = isSteppingDown;
     }
 
+    public LeaderInfo stepDown()
```

Comment:
A Java doc explaining the intention of this method would be good, since it is quite particular.